jd4/case.py

```diff
@@ -1,3 +1,5 @@
+import pyximport; pyximport.install()
```

Comment:
please don't use this in final version, as it increase startup time and memory footprint (it's also essential to keep the memory footprint small before forking workers). use static compilation (see setup.py). and why do you need to access cython module here?